### ISSUE TITLE
Runnen on same host as openbalena

### ISSUE THREAD

**Issue by RafSchandevyl** [dcaputo-harmoni/open-balena-admin#22]
Hi,
I'm trying to run openbalena-admin on the same host as openbalena itself.  it seems to start, but when i go to the url i get 503 service not available error.  could this be that de haproxy needs to be changed to be able to connect ?

i also get the certificate off the api.

**Comment by cakoolen** [comment]
I've now manually configured the ha proxy of the openbalena compose to include the services of the admin. After all that is working I now get `cannot Get /` from my browser with little hints on what's going wrong

Did you manage to get it working in the end?

**Comment by brynmathias** [comment]
@cakoolen do you have a snippet of your ha proxy?
I'm trying to do the same now

**Comment by cakoolen** [comment]
I just merged the config necessary for open-balena-admin with the haproxy present in the openbalena config. The drawback of that is that there are now hard dependencies between the two compose files. It would be a lot cleaner to merge the two all-together or as suggested above to deploy them separately. Adding the services to the openbalena haproxy ment that I had to tweak the compose file of the open-balena-admin repo as well.
